requirements_all.txt

```diff
@@ -404,10 +404,12 @@ iglo==1.1.3
 # homeassistant.components.ihc
 ihcsdk==2.1.1
 
-# homeassistant.components.influxdb
 # homeassistant.components.sensor.influxdb
 influxdb==4.1.1
```

Comment:
Should the sensor platform be upgraded at the same time?

Reply:
It should. Thanks for the feedback.